front/src/common-app/components/vote-options.component.tsx

```diff
@@ -16,6 +16,7 @@ export const VoteOptionsComponent: React.FC<Props> = props => {
   const { onVoteChosen, votedStatus, buttonFinishVoting } = props;
   const [voteChosen, setVoteChosen] = React.useState('');
   const [voteActive, setVoteActive] = React.useState('');
+  const elementFocus = React.useRef<HTMLDivElement>(null);
```

Comment:
rename to `yourVoteElement` and replace `HTMLDivElement` by the correct one

Reply:
Fix